buddypress/members/index-directory.php

```diff
@@ -361,16 +360,16 @@
 		continue;
 	}
 
-
+	
 	// Search / location / language!
 	if ( $search_user && false === $get_tag && $country_code && $language_code ) {
-		if ( $info['language']->display &&
+		if ( $info['languages']->display &&
```

Comment:
why are we changing the info key here from langauge to langauges?

Reply:
This was a typo. It's "languages" everywhere else